runtime/lua/vim/_defaults.lua

```diff
@@ -23,7 +23,11 @@ do
 
   vim.api.nvim_create_user_command('EditQuery', function(cmd)
     vim.treesitter.query.edit(cmd.fargs[1])
-  end, { desc = 'Edit treesitter query', nargs = '?' })
+  end, {
+    desc = 'Edit treesitter query',
+    nargs = '?',
+    complete = function(...) return vim.treesitter.language._complete(...) end,
```

Comment:
lint doesn't like this

```suggestion
    complete = function() return vim.treesitter.language._complete() end,
```